### ISSUE TITLE
Luminosity ratio is less than 4.5:1 for the 'Normal', Arial, and 15px texts present in Body edit field.

### ISSUE THREAD

**Issue by Ram-G** [Azure/LogicAppsUX#4456]
### Describe the issue 

1. Open designer to an action which uses HTML editor. 
2. Observe luminosity of toolbar buttons. 

Actual Result:

 Luminosity ratio is less than 4.5:1 for the 'Normal', Arial, and 15px texts present in Body edit field.

 Observation: Currently the ratio is 4.478:1.

Expected Result:

 Luminosity ratio for the 'Normal', Arial, and 15px texts present in Body edit field should be more than or equal to the required ratio of 4.5:1.

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

![image](<https://github.com/Azure/LogicAppsUX/assets/10837129/61959c8d-c559-4a2d-b494-97b9bf857d70>)

### Browser 

Edge

### Additional context 

ADO link: [https://msazure.visualstudio.com/One/\_workitems/edit/27260326](<https://msazure.visualstudio.com/One/_workitems/edit/27260326>)
